source/main.civet

```diff
@@ -203,7 +203,8 @@ makeCache := ({hits, trace}: CacheOptions = {}): CacheEvents ->
 
       return if uncacheable.has(ruleName)
 
-      key: CacheKey := [ruleName, state.pos, ...getStateKey()]
+      [stateKey, tagKey] := [...getStateKey()]
```

Comment:
I don't think we need the `[...]` wrapper here (also below).

I also wonder whether we could avoid a Set wrapper when the jsxTag is null, which is most of the time... Now that it's in front.